src/Akka.Hosting/Logging/LoggerFactoryLogger.cs

```diff
@@ -68,71 +62,19 @@ protected override bool Receive(object message)
         
         protected virtual void Log(LogEvent log, ActorPath path)
         {
-            var message = GetMessage(log.Message);
-            _akkaLogger.Log(GetLogLevel(log.LogLevel()), log.Cause, _messageFormat, GetArgs(log, path, message));
-        }
-
-        private static object[] GetArgs(LogEvent log, ActorPath path, object message)
-            => new []{ log.Timestamp, log.LogSource, path, log.Thread.ManagedThreadId, message };
-
-        private static object GetMessage(object obj)
-        {
-            try
-            {
-                return obj is LogMessage m ? string.Format(m.Format, m.Args) : obj;
-            }
-            catch (Exception ex)
-            {
-                // Formatting/ToString error handling
-                var sb = new StringBuilder("Exception while recording log: ")
-                    .Append(ex.Message)
-                    .Append(' ');
-                switch (obj)
-                {
-                    case LogMessage msg:
-                        var args = msg.Args.Select(o =>
-                        {
-                            try
-                            {
-                                return o.ToString();
-                            }
-                            catch(Exception e)
-                            {
-                                return $"{o.GetType()}.ToString() throws {e.GetType()}: {e.Message}";
-                            }
-                        });
-                        sb.Append($"Format: [{msg.Format}], Args: [{string.Join(",", args)}].");
-                        break;
-                    case string str:
-                        sb.Append($"Message: [{str}].");
-                        break;
-                    default:
-                        sb.Append($"Failed to invoke {obj.GetType()}.ToString().");
-                        break;
-                }
-
-                sb.AppendLine(" Please take a look at the logging call where this occurred and fix your format string.");
-                sb.Append(ex);
-                return sb.ToString();
-            }
+            _akkaLogger.Log<LogEvent>(GetLogLevel(log.LogLevel()), new EventId(), log, log.Cause, (@event, exception) => @event.ToString());
```

Comment:
We have a minimalist logging call now - it defers any type of formatting until the Microsoft.Extensions.Logging pipeline calls it. Uses the built-in `LogEvent.ToString` method that the rest of Akka.NET's loggers do.